EvalView/views.py

```diff
@@ -38,6 +40,8 @@
 LOGGER = _get_logger(name=__name__)
 
 # pylint: disable=C0103,C0330
+
```

Comment:
I think pylint annotations must be next to the function they apply to? Please check.